test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -60,6 +60,28 @@ Describe "Get-ChildItem" -Tags "CI" {
             $file.Count | Should be 1
             $file.Name | Should be "pagefile.sys"
         }
+        It "Should continue enumerating a directory when a contained item is deleted" -Skip:($IsWindows) {
```

Comment:
No. On Windows the test will fail because the enumeration will succeed with no errors emitted in the middle of the process, and `$Error.Count` will be zero.